pkg/pool-weighted/contracts/CustomFeeAuthorizer.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+
+import "@balancer-labs/v2-solidity-utils/contracts/openzeppelin/Ownable.sol";
+
+contract CustomFeeAuthorizer is Ownable {
```

Comment:
Won't this conflict with Balancer's own access control mechanism?

Reply:
I think it would make more sense instead to introduce the notion of `address public solver`; `solver` must be set on pool creation and can't be edited; `solver` can edit `isCustomFeeSetter` and call `setSwapFeePercentage`